ci-operator/step-registry/cucushift/upgrade/cucushift-upgrade-chain.yaml

```diff
@@ -2,6 +2,7 @@ chain:
   as: cucushift-upgrade
   steps:
   - ref: cucushift-pre
+  - ref: user-setting-htpasswd
```

Comment:
> That's weird! Not sure why this new step user-setting-htpasswd wasn't called

@jianzhangbjz for your this question in previous comment, seems it is due to line 4 and 5 should swap order?